engine/src/main/java/io/camunda/zeebe/engine/processing/resource/ResourceDeletionProcessor.java

```diff
@@ -130,6 +151,25 @@ private void deleteDecision(final PersistedDecision persistedDecision) {
     stateWriter.appendFollowUpEvent(keyGenerator.nextKey(), DecisionIntent.DELETED, decisionRecord);
   }
 
+  private void deleteProcess(final DeployedProcess process) {
+    // We don't add the checksum or resource in this event. The checksum is not easily available
+    // and the resources are left out to prevent exceeding the maximum batch size.
```

Comment:
🔧 I don't like this. For users, it can be very useful to have the resource of the deleted resource. I can also imagine that the checksum could be useful. 

❓ Is it really easy to exceed the maximum batch size this way? Are there so many records written by this processor?

❓ Why is it difficult to get the checksum here?

Reply:
> ❓ Is it really easy to exceed the maximum batch size this way? Are there so many records written by this processor?

It doesn't at the moment, but it will write more events later on when we add termination. I wonder what the value is for users to have the resource. If they are actively consuming records and are interested in the resource they will have it stored already because of the deployment. This means the key is sufficient to find the resource.

> ❓ Why is it difficult to get the checksum here?
We'd have to recalculate it, which as you saw when we paired proved to be troublesome.

Reply:
> If they are actively consuming records and are interested in the resource they will have it stored already because of the deployment.

That makes sense, but can we get it in the response to the client? There it may be useful.

Reply:
You mean the key?

Reply:
No the deleted resources. Similar to deploying

Reply:
We could, it still wouldn't contain the actual resources though. We don't return those for deployments either. I just really don't see the value of it at all.

Client send a request saying "Hey! Please delete the resource with this key (aka this process id and version)". We return a response "Yo! We will delete the resource with this key, process id and version". All the metadata is wat the client already knows when sending the request. So a "Yo! We will delete it, thank you!" would be sufficient imo.

But if it eases your mind I'm happy to add the process id, version and key to the response.

Reply:
Thanks @remcowesterhoud That makes total sense! 👍